### ISSUE TITLE
Change local storage location

### ISSUE THREAD

**Issue by iferres** [singularityhub/sregistry#423]
Hi, I had to change some configuration in my desktop and my `/export/home/iferres/.singularity` dir is now located at `/home/iferres/.singularity`. Now sregistry isn't able to reach the images (although they are there). How can I change the path sregistry is looking for the image files?
Thanks!

**Comment by vsoch** [comment]
Hey @iferres! First, moving this issue to sregistry (let me know if this is not correct - you seem to be asking about singularity registry server and not the client!)

So the images directory (that the registry sees) is a volume mount - it defaults to ./images on your local machine in the [docker compose](https://github.com/singularityhub/sregistry/blob/5a26a2495b98553ffa1a465af997e4ac9a444b56/docker-compose.yml#L17) so that's the line you should change if your images directory changes. That says I'm not sure why changing that location of the singularity cache would change the behavior, so maybe you were interested in the client. Either way - I can try to help! Can you give me a more concrete example of an error  / what you are trying to do?

**Comment by iferres** [comment]
Ops, sorry for the late response (I was out some days and then I forgot about this) and thank you very much. I have a conda installation of sregistry. I can't have docker running on my machine (employer policy). Is there any script where I can change this path?

**Comment by vsoch** [comment]
Can you clarify if you are using sregistry-cli or sregistry server?

**Comment by iferres** [comment]
Yep, the server. I have just make it work. I just deleted (sregistry rm) old images and pulled them again and now it works. Feel free to close this. Thank you!!

**Comment by vsoch** [comment]
So wait - you are using sregistry server without docker? Do I understand that correctly?

Definitely feel free to close it if you don't need help anymore - sorry for my confusion!
